deploy/chart/templates/daemonset.yaml

```diff
@@ -36,6 +36,15 @@ spec:
             {{- if .Values.linodegoDebug }}
             - --linodego-debug={{ .Values.linodegoDebug }}
             {{- end }}
+            {{- if .Values.routeController }}
+            - --enable-route-controller=true
+            - --vpc-name={{ .Values.routeController.vpcName }}
```

Comment:
nit: If vpc-name is required, we should verify that in the helm chart and error if its not provided

Reply:
Updated